src/mysky/encrypted_files.test.ts

```diff
@@ -132,20 +133,23 @@ describe("decryptJSONFile", () => {
     const data = fileData.slice(0, fileData.length - 1);
     expect(data.length).toEqual(4095);
     expect(() => decryptJSONFile(data, key)).toThrowError(
-      "Expected parameter 'data' to be padded encrypted data, length was '4095', was type 'object', value '174,167,134,21,46,207,180,245,44,139,11,69,252,151,172,83,91,4,3,35,8,124,58,113,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,216,246,27,114,110,131,41,62,175,48,210,216,97,136,48,8,79,76,33,85,100,240,252,91,188,153,39,76,217,94,167,68,133,137,216,208,244,253,19,105,85,177,254,135,12,20,24,85,185,165,5,14,89,243,15,157,237,128,66,76,41,181,192,187,199,218,199,82,43,134,154,161,91,215,191,119,33,42,7,137,188,71,228,251,245,222,30,193,...'"
+      "Expected input to be decrypted 'data' to be padded encrypted data, length was '4095'"
```

Comment:
This error is rather confusing. Is it expected that `data` is decrypted or padded encrypted data?

The previous version is more readable, aside from the raw bytes. The best of both worlds would be "Expected parameter 'data' to be padded encrypted data with length 4096 but length was 4095".